src/Aqua.jl

```diff
@@ -1,6 +1,13 @@
+"""
+    Aqua
```

Comment:
This docstring seems less useful than the readme to me

Reply:
I'll remove it and adapt the test